src/datachain/catalog/loader.py

```diff
@@ -100,27 +100,22 @@ def get_warehouse(in_memory: bool = False) -> "AbstractWarehouse":
     return warehouse_class(**warehouse_args)
 
 
-def get_distributed_class(**kwargs):
+def get_udf_distributor_class() -> type["AbstractUDFDistributor"]:
```

Comment:
Return class instead of class instance, since `run_worker` staticmethod is used [below](https://github.com/iterative/datachain/pull/997/files#diff-6ceb48583afbfb6331ecfabbf5ca8d3a19f0ebb63c3ac9b1e94d4ea33691d31eR97), no need to initialize instance.